### ISSUE TITLE
Is 1.13.1 a stable release?

### ISSUE THREAD

**Issue by ryandesign** [TigerVNC/tigervnc#1607]
Is 1.13.1 a stable release? I see that 1.13.1 was tagged a few days ago but the [latest release](https://github.com/TigerVNC/tigervnc/releases/latest) is still 1.13.0.

**Comment by CendioOssman** [comment]
Yes. It takes some time for us to get all the builds up. Everything should be in place now, though.
